src/components/app/details/appDetails/utils.tsx

```diff
@@ -341,6 +363,12 @@ export function addQueryParamToGrafanaURL(url: string, appId: string | number, e
             url += (statusCode.includes("xx")) ? `&var-response_code=` : `&var-response_code=${statusCode}`;
         }
     }
+    if (chartName === 'latency') {
+        if (!isNaN(latency) ) {
+            latency = latency/100;
```

Comment:
should it be - 
latency = (latency/100).toFixed(3);
as (99.9/100) gives 0.9990000000000001
?

Reply:
user can give more digits for eg 99.999 which would depend on SLA company follows